packages/playground/src/pages/Components/Components.js

```diff
@@ -54,6 +54,17 @@ getOriginKey()
             }
         };
 
+        window.bacsdd = checkout
+            .create('directdebit_GB', {
+                data: {
+                    holderName: 'Philip Dog',
+                    bankAccountNumber: '12345678',
+                    bankLocationId: '123456',
+                    shopperEmail: 'phil@ddog.co.uk'
+                }
+            })
+            .mount('.bacsdd-field');
```

Comment:
Since we moved it to "Vouchers" maybe we can remove it from here.

Reply:
Except then you can't rest the full flow - 'cos you can't submit from Vouchers

Reply:
removed form components playground